app/controllers/items_controller.rb

```diff
@@ -1,6 +1,7 @@
 class ItemsController < ApplicationController
   def index
     @items = Item.where(category_id: params[:category])
```

Comment:
You should refactor to `@items = @category.items` and switch the order of the lines in this method.